docs/iot/index.yml

```diff
@@ -45,7 +45,7 @@ landingContent:
       - linkListType: reference
         links:
           - text: API Reference 
-            url: /dotnet/api?view=iot-dotnet-1.1
+            url: ../../api/index.md?view=iot-dotnet-1.1
```

Comment:
I'm surprised this is a valid link. Is it validated by the build?

Reply:
It's actually in the repo `https://github.com/dotnet/docs/blob/master/api/index.md`, so yes, it's valid and validated.